src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -33,6 +33,17 @@
 #define KEY_AS(name) !strcmp(key, name)
 
 
+static bool _curve2line(Point& a, Point& b, Point& c, Point& d)
+{
+    auto m = (d.y - a.y) / (d.x - a.x);
+    auto by = m * (b.x - a.x) + a.y;
+    if (!mathEqual(by, b.y)) return false;
+    auto cy = m * (c.x - a.x) + a.y;
+    if (!mathEqual(cy, c.y)) return false;    
+    return true;
+}
```

Comment:
my propositioin:
```
static bool _isBetween(const Point& check, const Point& a, const Point& b)
{
    return check.x >= mathMin(a.x, b.x) && check.x <= mathMax(a.x, b.x) &&
           check.y >= mathMin(a.y, b.y) && check.y <= mathMax(a.y, b.y);
}

static bool _curve2line(const Point& a, const Point& b, const Point& c, const Point& d)
{
    //vertical line
    if (mathEqual(a.x, d.x) && mathEqual(a.x, c.x) && mathEqual(a.x, b.x)) {
        return _isBetween(b, a, d) && _isBetween(c, a, d);
    }

    auto m = (d.y - a.y) / (d.x - a.x);
    auto by = m * (b.x - a.x) + a.y;
    if (!mathEqual(by, b.y)) return false;
    auto cy = m * (c.x - a.x) + a.y;
    if (!mathEqual(cy, c.y)) return false;
    
    return _isBetween(b, a, d) && _isBetween(c, a, d);
}
```

tvgMath.h has to be included

ex. in the 'world_locations.json' happens that `_isBetween` returns false